src/gpl/src/nesterovPlace.cpp

```diff
@@ -875,6 +877,9 @@ float NesterovPlace::getStepLength(
   debugPrint(
       log_, GPL, "getStepLength", 1, "GradientDistance: {:g}", gradDistance);
 
+  if (gradDistance == 0) {
+    return 0;
+  }
```

Comment:
You are still blocking the isinf call by having this code.

Reply:
shall I remove that if block?

Reply:
yes

Reply:
If I remove `if` block, not resolving this [testcase](https://github.com/The-OpenROAD-Project/OpenROAD/pull/2370#issuecomment-1286370120)